pants_plugins/sendwave/pants_node/package.py

```diff
@@ -125,7 +125,7 @@ async def get_node_package_digest(field_set: NodeProjectFieldSet) -> Digest:
         ),
     )
     logger.debug(proc.stdout)
-    if field_set.output_path:
+    if field_set.output_path and field_set.output_path.value is not None:
```

Comment:
Ditto here: While testing https://github.com/waveremit/remit-srv/pull/9375 I was getting errors indicating that `NodeProjectFieldSet.output_path` can have a `value` of `None` while the object itself is still truthy, which would cause the `AddPrefix` function to complain that it can't convert a NoneType to a string.